packages/runtime/src/extensibility/facades/transport/TagsFacade.ts

```diff
@@ -0,0 +1,12 @@
+import { Result } from "@js-soft/ts-utils";
+import { BackboneGetTag } from "@nmshd/transport";
```

Comment:
this is very lazy, why do you use a transport type instead of creating a DTO like everywhere else?

Reply:
Is the mapper enough, or should I add a serval class ?